priv/templates/phx.gen.auth/migration.ex

```diff
@@ -8,7 +8,8 @@ defmodule <%= inspect schema.repo %>.Migrations.Create<%= Macro.camelize(schema.
 <%= if schema.binary_id do %>      add :id, :binary_id, primary_key: true
 <% end %>      <%= migration.column_definitions[:email] %>
       add :hashed_password, :string, null: false
-      add :confirmed_at, :naive_datetime
+      add :confirmed_at, <%= inspect schema.timestamp_type %>
```

Comment:
Would output one of `:naive_datetime`, `:utc_datetime` or `:utc_datetime_usec`.

I understand from the first note in https://hexdocs.pm/ecto/Ecto.Schema.html#module-primitive-types that those are valid values both in the schema definition and in the migration file.

> When using database migrations provided by "Ecto SQL", you can pass your Ecto type as the column type. However, note the same Ecto type may support multiple database types. For example, all of :varchar, :text, :bytea, etc. translate to Ecto's :string. Similarly, Ecto's :decimal can be used for :numeric and other database types. For more information, see [all migration types](https://hexdocs.pm/ecto_sql/Ecto.Migration.html#module-field-types).